src/API/Plugin.php

```diff
@@ -11,7 +11,16 @@ class Plugin extends Client
     const SETTING_DEFAULT_SETTINGS = 'default_settings';
     const SETTING_IP_REWRITE = 'ip_rewrite';
     const SETTING_PROTOCOL_REWRITE = 'protocol_rewrite';
-    const PLUGIN_SPECIFIC_CACHE = 'plugin_specific_cache';
+    const SETTING_PLUGIN_SPECIFIC_CACHE = 'plugin_specific_cache';
+
+    public static function getPluginSettingsKeys() {
+        return array(
+            self::SETTING_DEFAULT_SETTINGS,
+            self::SETTING_IP_REWRITE,
+            self::SETTING_PROTOCOL_REWRITE,
+            self::SETTING_PLUGIN_SPECIFIC_CACHE
+        );
+    }
```

Comment:
PluginActions are stored in cloudflare/cloudflare-plugin-backend now too.  I moved all the logic into the backend for precisely that reason, if we add a new plugin route to any of our integrations the rest of the plugins need to be able to get it next time they upgrade the backend.

At a high level all this api does is call the `dataStore` so as long as the keys we use to store the settings are unique we can use them across all the plugins without any plugin specific code.  Anytime we do need plugin specific code we'll provide an abstract method for them to override in their implementation like we did with default settings.